Objects/regions.c

```diff
@@ -103,6 +129,10 @@ PyObject* error_tgt = Py_None;
 // Once an error has occurred this is used to surpress further checking
 bool error_occurred = false;
```

Comment:
Something I've been curious about is why this is here instead of using [PyErr_Occurred](https://docs.python.org/3/c-api/exceptions.html#c.PyErr_Occurred). Seems like needless double bookkeeping unless I'm missing something.